sentry_streams/sentry_streams/adapters/arroyo/rust_arroyo.py

```diff
@@ -211,11 +244,19 @@ def sink(self, step: Sink[Any], stream: Route) -> Route:
                 bucket = step.bucket
                 thread_count = 1
 
-            object_generator = step.object_generator
+            # TODO: This object_generator is just used to get the name of the object that is being written.
+            # Fix this to wrap the actual step instead of just the object_generator.
+            # This will at least capture the number of calls to the step, if nothing else.
+            def wrapped_generator() -> str:
+                start_time = input_metrics(step.name, None)
```

Comment:
I'd juse remove the metrics now and just leave the TODO here. So we do not create confusion on what works